docs/guides/supported_backends.md

```diff
@@ -0,0 +1,51 @@
+# Supported Backends 
+
+
+GuideLLM requires an OpenAI-compatible server to run evaluations. [vLLM](https://github.com/vllm-project/vllm) is recommended for this purpose; however, GuideLLM is compatible with many backend inference servers such as TGI, llama.cpp, and DeepSparse. 
+
+## OpenAI/HTTP Backends
+
+### Text Generation Inference 
+[Text Generation Inference](https://github.com/huggingface/text-generation-inference) can be used with GuideLLM. To start a TGI server with a Llama 3.1 8B using docker, run the following command:
+```bash
+docker run --gpus 1 -ti --shm-size 1g --ipc=host --rm -p 8080:80 \
+  -e MODEL_ID=https://huggingface.co/llhf/Meta-Llama-3.1-8B-Instruct \
```

Comment:
Not sure how reasonable this request is, but the different examples use different model quantizations. Would be awesome if they were the same:)